Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -891,6 +892,30 @@ int CLuaElementDefs::IsElementWithinMarker ( lua_State* luaVM )
     return 1;
 }
 
+int CLuaElementDefs::GetColShapeType ( lua_State* luaVM )
+{
+    // Verify the arguments
+    CClientColShape* pColShape = NULL;
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pColShape );
+
+    if ( ! argStream.HasErrors ( ) )
+    {
+        // Grab our VM
+        CLuaMain* pLuaMain = m_pLuaManager->GetVirtualMachine ( luaVM );
+        if (pLuaMain)
+        {
+            lua_pushnumber ( luaVM, pColShape->GetShapeType ( ) + 1 );
```

Comment:
Any specific reason not to use the 0 value here?

Reply:
becose lua start index from 1, not 0 like c++ and other languages

Reply:
But in this case it's not an index